Lab Rat - GWJ 2020/Scenes/Level/level-jonathan/level-jonathan.gd

```diff
@@ -0,0 +1,100 @@
+extends Spatial
+
+
+var laser_count = 0
+
+var mission_archieved = [] #hier wird gespeichert, welche laser schon "geöfnet" wurden
+var last_laser_went_into = 0
+var current_mission = 1
+var highest_mission = 1
+
+var mission_to_do = []
+
+var possible_laser = []
+
+
+var gerade_vorspiel = false
+var takt_im_vorspiel = 1
+var laenge_des_vorspiels = 1
```

Comment:
😅

Reply:
die variablen sind nicht unötig! die verändern sich schon noch. Aber ja, ich bin nicht soo kreativ im Namenfinden 😅

Reply:
Geht eigentlich kein "ä" im GDScript-Namen?

Reply:
man nimmt im allgemeinen keine umlaute, das hat doch eh nur die deutsche sprache, oder?

Reply:
glaube ja. ich nehm zumindest nie. bei python geht glaub ich zum beispiel nicht, bei C auch nicht und jetzt hab ich mir es halt schon so angewöhnt ohne umlaute zu schreiben

Reply:
Ok, sonst könnte man schreiben "VorspielLen" oder "vorspiel_length"

Reply:
man könnte viel schreiben. bist du denn mit deinem level schon fertig? Mach das doch bitte zuerst. Ich werde irgendeinen irrelevanten namen irgendeiner Variablen jetzt eh nicht mehr verändern, nur weil du meinst dass man die variable auch anders nennen könnte!

Reply:
"ae" ist einfach ziemlich hässlich. Außerdem hatte ich gerade die Aufgabe des Pausenmenüs erledigt, als nächstes kommt noch das interaktive Laden. Wenn dann noch Zeit ist, kann ich auch ein Level machen.

Reply:
Also "haesslich" lol

Reply:
meinst du schaffst das interaktive laden? ich würde das nämlich glaub ich nicht mehr. wenn nicht, würde ich nämlich gleich mit einem Level anfangen, als jetzt stundenlang irgendetwas zu versuchen, was du dann gar nicht fertigbringst

Reply:
Ja, denke schon, so schwer ist das nicht. Hatte das ja schon in einem alten Projekt gemacht, und viel geändert hat sich da nicht.

Reply:
Ja, denke schon, so schwer ist das nicht. Hatte das ja schon in einem alten Projekt gemacht, und viel geändert hat sich da nicht.